### ISSUE TITLE
conversion to async function buggy with named exported functions

### ISSUE THREAD

**Issue by targos** [Microsoft/TypeScript#27429]
**TypeScript Version:**  3.1.1

**Search Terms:** async export convert 

**Code**

```ts
export function testAsync() {
  return apiFetchJSON('url').then((data) => {
    if (!data.error) {
      return data;
    } else {
      return null;
    }
  });
}
```

**Expected behavior:**

Converted to `export async function testAsync() { ... }`

**Actual behavior:**

Converted to `async export function testAsync() { ... }`

**Comment by DanielRosenwasser** [comment]
@uniqueiniquity can you port this over to `release-3.1`?

**Comment by DanielRosenwasser** [comment]
3.1 fix is in 7c6a14e65df2240af135f5a9fdd235ebb28b4d50
